EIPS/eip-1.md

```diff
@@ -52,7 +55,7 @@ Each status change is requested by the EIP author and reviewed by the EIP editor
 * **Active** -- Some Informational and Process EIPs may also have a status of “Active” if they are never meant to be completed. E.g. EIP 1 (this EIP).
 * **Work in progress (WIP)** -- Once the champion has asked the Ethereum community whether an idea has any chance of support, they will write a draft EIP as a [pull request]. Consider including an implementation if this will aid people in studying the EIP.
   * :arrow_right: Draft -- If agreeable, EIP editor will assign the EIP a number (generally the issue or PR number related to the EIP) and merge your pull request. The EIP editor will not unreasonably deny an EIP.
-  * :x: Draft -- Reasons for denying draft status include being too unfocused, too broad, duplication of effort, being technically unsound, not providing proper motivation or addressing backwards compatibility, or not in keeping with the [Ethereum philosophy](https://github.com/ethereum/wiki/wiki/White-Paper#philosophy).
+  * :x: Draft -- Reasons for denying draft status include being too unfocused, too broad, duplication of effort, not providing proper motivation or addressing backwards compatibility, or not in keeping with the [Ethereum philosophy](https://github.com/ethereum/wiki/wiki/White-Paper#philosophy).
```

Comment:
I don't see the need to remove this reason. I agree with seeing an EIP merged as Draft as soon as possible, but there are still standards like when a technical implementation makes no sense.

Reply:
@nicksavers The issue is that it puts more burden on editors, and encourages more discussion of a draft before it has a stable URL. IMO, we should be merging as soon as possible, as that way it has a stable URL and one place for discussions to go, instead of being fragmented across EIPs.

Reply:
I agree with @nicksavers. Editors should make sure an EIP can be implemented as it is specified. What editors should refrain on is bikeshedding on multiple implementation/specification possibilities.
Anyone can just publish an EIP with the correct syntax but nonsense technical content.

Reply:
Maybe it is time to introduce another stage for EIPs? Draft, Complete, Accepted / Rejected

I agree that an EIP in draft stage should have a stable URL and there should be a way to gauge interest before writing a full specification. But I also have problems reading EIPs that clearly have gaps or even worse, being asked to state my opinion on them.

Reply:
@chriseth What would the semantics of this new stage be? It seems like it'd be the same as draft, the only difference is that the author is asking you to review it.

Reply:
I see **Final Call** as the stage in which the author signals that the proposal should not be in Draft anymore.